JavaFXUserInterface/src/main/java/external/ModelDAO.java

```diff
@@ -8,22 +8,28 @@
 package external;
 
 import app.item.Model;
+import app.item.TrainedModel;
+import rul.models.ModelStrategy;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
 import java.util.List;
 
 public interface ModelDAO {
 
     String getModelNameFromAssetTypeID(String assetTypeId);
 
-    int getModelIDFromAssetTypeID(String assetTypeID);
-
-    void updateRMSE(Double rmse, int modelId, int assetTypeId);
+    TrainedModel createTrainedModelFromResultSet(ResultSet rs, boolean withModel) throws SQLException;
```

Comment:
This method is never used. Why did you add it?

Reply:
getModelEvaluation?